sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -41,6 +41,22 @@ public partial class ClientResult<T> : System.ClientModel.ClientResult
         public static implicit operator T (System.ClientModel.ClientResult<T> result) { throw null; }
     }
 }
+namespace System.ClientModel.Options
```

Comment:
I don't think we'll want to introduce a new namespace for Options types -- they're probably best in System.ClientModel.Primitives with other options types?